data/projects/optimize_caching.md

```diff
@@ -1,28 +1,46 @@
-This project involves improving coala's performance by implementing caching
-strategies. One example is to use git-like approach at file-collection level
-to not explore directories that haven't been modified. Caching per section can
-also be implemented so sections without issues don't rerun on all files where
-issues in other sections occurred.
+Currently, coala supports caching of project files. This was a big performance
+improvement over running code analysis for all files, even if most files
```

Comment:
this sentence sounds not right to me...
maybe "With it, coala only runs on files that changes since the last run, potentially giving huge performance improvements compared to simply running coala on everything regardless."